### PR TITLE
ISPN-15311 Listener converter in SUBSCRIBE must be externalizable

### PR DESCRIPTION
**Pull request by rigazilla**
https://issues.redhat.com/browse/ISPN-15311

### CODE REVIEW

File: server/resp/src/main/java/org/infinispan/server/resp/filter/EventListenerConverter.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.infinispan.commons.marshall.AdvancedExternalizer;
+import org.infinispan.encoding.DataConversion;
+import org.infinispan.metadata.Metadata;
+import org.infinispan.notifications.cachelistener.filter.CacheEventConverter;
+import org.infinispan.notifications.cachelistener.filter.EventType;
+import org.infinispan.server.resp.ExternalizerIds;
+
+public class EventListenerConverter<K, V, C> implements CacheEventConverter<K, V, C> {
+   public static AdvancedExternalizer<EventListenerConverter> EXTERNALIZER = new EventListenerConverter.Externalizer();
+   private DataConversion dc;
```
Comment (jabolina) [inline server/resp/src/main/java/org/infinispan/server/resp/filter/EventListenerConverter.java:20]: Although this works, the `DataConversion` needs to be initialized to handle the parsing :(. Looking at the `CacheNotifierImpl` (https://github.com/infinispan/infinispan/blob/main/core/src/main/java/org/infinispan/notifications/cachelistener/CacheNotifierImpl.java#L1605-L1619), unsure if just having a method here annotated with `@Inject` would do the trick?

**Review by jabolina — APPROVED** [review]
LGTM. Is something still needed?

**Comment by rigazilla** [comment]
> LGTM. Is something still needed?

imo it can be merged

**Comment by jabolina** [comment]
Thanks, @rigazilla!

**Pull request merged**